libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -47,12 +47,14 @@ class AP_Frsky_Telem {
     static bool set_telem_data(const uint8_t frame,const uint16_t appid, const uint32_t data);
 #endif
 
+#if HAL_GCS_ENABLED
```

Comment:
we should probably disable RC telem if GCS is disabled, send_textv() will be compiled out anyway so that worthless to try compile RC telem